doc/experiment.rst

```diff
@@ -1,327 +1,423 @@
+configuration and can address the cluster with a SmartRedis client
+like a single block of memory using simple put/get semantics in SmartRedis.
+The database shards communicate with each other via TCP/IP in the driver script and application.
+SmartRedis will ensure that data is evenly distributed among all nodes in the cluster.
+
+Clustered Deployment Diagram
+""""""""""""""""""""""""""""
+During clustered deployment, a SmartSim ``Model`` (the application) runs on separate
+compute node(s) from the database node(s).
+A clustered database is optimal for high data throughput scenarios
+such as online analysis, training and processing.
+
+Below is an image illustrating communication
+between a clustered orchestrator and a
+multi-node model. In the Diagram, an instance of the application is
+running on each application compute node. A single SmartRedis client is initialized with
+the clustered database address and used to communicate with the application's compute nodes.
+Data is streamed from the application compute nodes to the sharded database via the client.
+
+.. figure::  images/clustered-orc-diagram.png
+
+Initialize a Clustered Orchestrator
+"""""""""""""""""""""""""""""""""""
+To create an ``Orchestrator`` that does not share compute resources with other
+SmartSim entities, use the ``Experiment.create_database()`` factory method.
+Specifying the parameter `db_nodes` as greater than or equal to 1 will determine
+whether your database is multi-sharded or single-sharded.
+This factory method returns an initialized ``Orchestrator`` object that
+gives you access to functions associated with the :ref:`Orchestrator API<orc_api>`.
+
+Colocated Orchestrator
+^^^^^^^^^^^^^^^^^^^^^^
+An ``Orchestrator`` can be created to share the compute node(s)
+and resources with a SmartSim ``Model``. In this case, the database
+is deployed on the same compute hosts as a Model instance
+defined by the user. In this deployment, the database is not connected
+together in a cluster and each shard of the database is addressed
+individually by the processes running on that compute host.
+If the SmartSim ``Model`` spans more than one
+compute node, the colocated database will also span all of the
+compute nodes. The colocated deployment strategy for the Orchestrator
+is ideal for use cases where a SmartSim ``Model`` is run on a compute node
+that has hardware accelerators (e.g. GPUs) and low-latency inference is
+a critical component of the workflow.
+
+Colocated Deployment Diagram
+""""""""""""""""""""""""""""
+During colocated deployment, a SmartSim orchestrator (the database) runs on the same
+compute node(s) as a Smartsim model (the application).
+This type of deployment is optimal for high data inference scenarios.
+
+Below is an image illustrating communication
+between a colocated model spanning multiple compute nodes, and the database
+running on each application compute node. A single SmartRedis client is initialized
```

Comment:
To the application, nothing really changes between the standard and co-located orchestrator. Each rank of the simulation connects only to the database that exists on the same node. So for example, if you have an application that parallelized over 128 cores on 4 nodes: you have 4 separate databases (not shards), 32 clients on each node, each of which is only connected to the database that is on the same compute node.